EDDI/EDDI.cs

```diff
@@ -1,32 +1,30 @@
-﻿using EddiCompanionAppService;
+﻿using Eddi;
+using EddiCompanionAppService;
 using EddiDataDefinitions;
 using EddiDataProviderService;
 using EddiEvents;
 using EddiSpeechService;
-using EddiStarMapService;
 using Newtonsoft.Json;
 using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.IO;
 using System.Linq;
 using System.Reflection;
 using System.Runtime.InteropServices;
-using System.Security.Permissions;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using System.Windows;
 using Utilities;
 
-namespace Eddi
+namespace EddiCore
 {
-    /// <summary>
-    /// Eddi is the controller for all EDDI operations.  Its job is to retain the state of the objects such as the commander, the current system, etc.
-    /// and keep them up-to-date with changes that occur.  It also passes on messages to responders to handle as required.
-    /// </summary>
-    public class EDDI
+	/// <summary>
+	/// Eddi is the controller for all EDDI operations.  Its job is to retain the state of the objects such as the commander, the current system, etc.
+	/// and keep them up-to-date with changes that occur.  It also passes on messages to responders to handle as required.
```

Comment:
It acts as the switchboard for passing events through all parts of the application including both responders and monitors. :-)

Reply:
Amended.